barretenberg/cpp/src/barretenberg/world_state/world_state.test.cpp

```diff
@@ -79,6 +81,10 @@ void assert_leaf_index(
     const WorldState& ws, WorldStateRevision revision, MerkleTreeId tree_id, const Leaf& value, index_t expected_index)
 {
     std::optional<index_t> index = ws.find_leaf_index<Leaf>(revision, tree_id, value);
+    EXPECT_TRUE(index.has_value());
+    if (!index.has_value()) {
+        return;
+    }
```

Comment:
Is the `if` only to prevent the linter from complaining about "unchecked optional access"?  (I think the `EXPECT_TRUE` above it will throw before we get to the if).